apps/sel4test-tests/arch/riscv/arch_frame_type.h

```diff
@@ -11,9 +11,12 @@
 
 /* This list must be ordered by size - highest first */
 static const frame_type_t frame_types[] = {
-    /* Rocket-Chip for zedboard only has 256MiB of RAM, so we can't allocate a 1GiB page */
+    /* Rocket-Chip for zedboard and base Rocket-Chip only have 256MiB of RAM, 
+     * so we can't allocate a 1GiB page */
     /* Polarfire has 1GiB of memory can't allocate a 1GiB page for user space */
-#if __riscv_xlen == 64 && !defined(CONFIG_BUILD_ROCKET_CHIP_ZEDBOARD) && !defined(CONFIG_PLAT_ARIANE) &&!defined(CONFIG_PLAT_POLARFIRE)
+#if __riscv_xlen == 64 && !defined(CONFIG_BUILD_ROCKET_CHIP_ZEDBOARD) \
```

Comment:
Can't we remove `CONFIG_BUILD_ROCKET_CHIP_ZEDBOARD` then and just say this applies for all Rocket Chips? On the mailing list I have raised the question what the genera state of the Rocket Chip platform is, given that the repos on github indicate it has been abandonend.